test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.cs

```diff
@@ -148,6 +154,92 @@ public async Task CheckEnrichmentWhenSampling()
         await CheckEnrichment(new AlwaysOnSampler(), true, this.url).ConfigureAwait(false);
     }
 
+#if NET8_0_OR_GREATER
+    [Theory]
+    [MemberData(nameof(TestData))]
+    public async Task ValidateNet8MetricsAsync(HttpTestData.HttpOutTestCase tc)
+    {
+        var metrics = new List<Metric>();
+        var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddHttpClientInstrumentation()
+            .AddInMemoryExporter(metrics)
+            .Build();
+
+        var testUrl = HttpTestData.NormalizeValues(tc.Url, this.host, this.port);
+
+        try
+        {
+            using var c = new HttpClient();
+            using var request = new HttpRequestMessage
+            {
+                RequestUri = new Uri(testUrl),
+                Method = new HttpMethod(tc.Method),
+                Version = new Version(2, 0),
```

Comment:
Why is this needed?

Reply:
THis was related to https://github.com/open-telemetry/opentelemetry-dotnet/issues/4928. I have fixed this now.

Reply:
So is this needed?